concepticondata/concepticon.tsv

```diff
@@ -3174,3 +3174,55 @@ ID	GLOSS	SEMANTICFIELD	DEFINITION	ONTOLOGICAL_CATEGORY	REPLACEMENT_ID
 3179	CORAL ROCK	The physical world	Limestone created from the skeleton of reef-forming stony corals (Scleractinia), displaying their growth patterns.	Person/Thing	
 3180	SHOOT (WITH SLINGSHOT)	Warfare and hunting	Use a rubber-based projectile weapon to propel projectiles against a target.	Action/Process	
 3181	MUSTARD	Food and drink	It can be a yellow sauce with a strong taste or a plant which is used to make mustard sauce.	Person/Thing	
+3182	MARRIAGE OR WEDDING	Kinship	The ceremony of marrying someone or the state of being married.	Person/Thing	
+3183	BREATH OR BREATHE	The body	The air that goes into one's lungs or the act of breathing	Other	
+3184	BE ALIVE OR LIFE	The body	The act of being alive or the life itself.	Other	
+3185	BE DEAD OR DIE OR KILL	The body	To be not alive or to cease to live or to cause the end of a life.	Other	
+3186	BE DEAD OR DEATH OR DIE	The body	To be not alive or the end of a life or to cease to live.	Other	
+3187	DROWNED	The body	To have perished by suffocation in water.	Property	
+3188	MIGHTY OR POWERFUL OR STRONG	Emotions and values	Holding much (political, social, ...) power or being done using a huge amount of energy or having great physical strength.	Property	
+3189	STAB TO DEATH	The body	To kill by stabbing with a knife.	Action/Process	
+3190	BROKEN (NOT WORKING)	Basic actions and technology	Not capable of working.	Property	
+3191	BROKEN (SHATTERED)	Basic actions and technology	Broken into pieces.	Property	
```

Comment:
But this refers only to objects, correct? If I remember correctly, in the Tanzanian languages the examples for BROKEN seem to come from "broken bone"; I like the definitions we have now (certainly better than the confusion of all the previous "brokens"), but I can't really see what would map to "fractured". This one only referes to "fragmented", in a sense.

Reply:
I would not make a difference between BROKEN (SHATTERED) and "fragmented".

Reply:
Yes, neither do I! My suggestion would be to have it as BROKEN (SHATTERED OR FRACTURED), as fractured implies that something can be fixes/still used.

Reply:
Seems a bit long as a concept name, though.